tests/foreman/cli/test_host.py

```diff
@@ -1907,6 +1908,39 @@ def test_positive_erratum_applicability(katello_host_tools_client):
     assert settings.repos.yum_6.errata[2] not in applicable_erratum_ids
 
 
+@pytest.mark.katello_host_tools
+@pytest.mark.tier3
+def test_positive_apply_security_erratum(katello_host_tools_client):
+    """Apply security erratum to a host
+
+    :id: 4d1095c8-d354-42ac-af44-adf6dbb46deb
+
+    :expectedresults: erratum is recognized by the
+        `yum update --security` command on client
+
+    :CaseLevel: System
+
```

Comment:
please let me know what you think is the test case importance for this test. 
If you want you can update it here. Non blocking of course :)

Reply:
@tstrych Yes, test case importance for this test should be `High`, as similar it was in the previous module, and as you've already defined `:CaseImportance` at a module level, so it should be fine I believe, but if you would ask, so I can update same at test level as well.

Reply:
no you don't have to :) thanks for your opinion